### PR TITLE
fix links to point to ArduPilot supported cmds

### PR DESCRIPTION
**Pull request by Hwurzburg**
fixs #3071

### CODE REVIEW

File: common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ Overview
 ========
 
 The MAVLink protocol defines a large number of
-`MAV_CMD <https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml#L531>`__
```
Comment (rmackay9) [inline common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst:17]: We should update the line number from 531 to 1008

File: common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst [hunk 2]
```diff
@@ -98,7 +98,7 @@ defined in a terrain database).
 .. note::
 
    The other frame types defined in the MAVLink protocol (see
-   `MAV_FRAME <https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml#L434>`__)
+   `MAV_FRAME <https://github.com/ArduPilot/mavlink/blob/master/message_definitions/v1.0/common.xml#L434>`__)
```
Comment (rmackay9) [inline common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst:101]: This line number should be updated from 434 to 795.  These line numbers are all a bit fragile but..

Comment (Hwurzburg) [reply common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst]: fixed

File: common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst:17]: fixed

**Comment by Hwurzburg** [comment]
thanks for the review! fixed and merging...

**Pull request merged**